### ISSUE TITLE
Suggestion: Use fork of cleanflight-configurator

### ISSUE THREAD

**Issue by mikeller** [airfield/betaflight-toolbox#3]
Guys, before we start putting a lot of development effort into this, I think we should consider using a proper fork of cleanflight-configurator as the base of this project. This would have the following advantages:

- any improvements made to cleanflight-configurator can be easily cherry-picked into betaflight-configurator
- any improvements we do can be easily cherry-picked and contributed back into cleanflight-configurator

Are there any advantages to instead be using something that is a copy of cleanflight configurator?

(Also, do github rules even allow for re-submitting something as a 'new' project if it is a blatant copy of an existing project?)

**Comment by skaman82** [comment]
Theoretically this is a great idea. But there is one disadvantage (or I am doing this wrong): github just allows to fork one repo. As im my case, I have a fork of CF Configurator that I am working on and github don't allow me to create another fork of this.

**Comment by mikeller** [comment]
@skaman82: I think this is just the way that github is naming things. In terms of functionality, forks are more or less identical to branches. They call it a 'fork' if it goes from one account to another, and a 'branch' if it stays within the same account. So, in order to keep your own work separate, the best way is probably to fork CF configurator into your account, use the fork as the 'betaflight - configurator' release branch, and do your own work in a branch off of this.

**Comment by skaman82** [comment]
just created this fork: https://github.com/airfield/betaflight-configurator

**Comment by skaman82** [comment]
But still can not fork this to work on it, since i am working on the original CF Configurator in parallel :(

**Comment by mikeller** [comment]
I don't think you need to fork it. Just create a branch that pulls from CF configurator to do your work on that one, and create branches that pull from betaflight-configurator to do work on that one.
